packages/kotti-ui/source/kotti-comment/KtCommentInput.vue

```diff
@@ -59,10 +59,10 @@ export default defineComponent<KottiCommentInput.PropsInternal>({
 				emit('submit', {
 					message: text.value,
 					replyToUserId: props.replyToUserId,
-					parentId: props.parentId,
+					parentId: props.parentId ?? null,
 				})
 				text.value = null
-				textarea.value.style.height = '1.2rem'
+				if (textarea.value !== null) textarea.value.style.height = '1.2rem'
```

Comment:
I'd rather throw and error. This shouldn't actually happen at runtime unless I'm misunderstanding

Reply:
this is actually dead

Reply:
i also remember leaving a comment on another `KtComment` refactor for this and actually testing it, and it proved dead

but I'd reverify the testing